code/edgeidentity/src/test/java/com/adobe/marketing/mobile/edge/identity/IdentityPropertiesTests.java

```diff
@@ -256,6 +279,10 @@ public void test_primaryECIDIsAlwaysTheFirstElement() {
 		assertEquals("primaryAgain", flatMap.get("identityMap.ECID[0].id"));
 		assertEquals("secondary", flatMap.get("identityMap.ECID[1].id"));
 	}
+	// ======================================================================================================================
+	// Tests for method : setAdId(final String newAdId)
```

Comment:
Do you need to add test cases here or are they covered in IdentityExtensionTests?